EasyLearn/config/database.yml

```diff
@@ -4,19 +4,19 @@
 postgresql: &postgresql
   adapter: postgresql
   host: localhost
-  username: username
-  password: password
+  username: p_final
+  password: p_finalBC2021
   pool: 5
   timeout: 5000
 development:
   <<: *postgresql
-  database: db/p_final_development
+  database: p_final_development
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *postgresql
-  database: db/p_final_test
+  database: p_final_test
 production:
   <<: *postgresql
-  database: db/p_final_production
\ No newline at end of file
+  database: p_final_production
```

Comment:
Remember guys this is your file with your database credentials and it should stay in your git ignore and upload your only database.yml.example file

Reply:
Done.

Reply:
sure?
i can see the file in this PR yet

Reply:
Updated.

Reply:
nice, now, update the `database.yml.example`

Reply:
Done.

Reply:
Nice, now you can put resolve this conversation and take your LGTM good boy!!